transactions/src/main/java/org/jboss/as/txn/service/UserTransactionAccessControlService.java

```diff
@@ -21,24 +25,25 @@
  *
  * @author Jaikiran Pai
  * @author Eduardo Martins
+ * @author <a href="mailto:ropalka@redhat.com">Richard Opalka</a>
  */
-public class UserTransactionAccessControlService implements Service<UserTransactionAccessControlService> {
-
+public class UserTransactionAccessControlService implements Service {
```

Comment:
Can this one not implement Service at all?  Use something like org.jboss.msc.SimpleService as the Service implementation. I don't think dependents are meant to have access to start/stop.

Reply:
In this particular case it must implement this interface because e.g. org.jboss.as.ejb3.subsystem.EJB3UserTransactionAccessControlService injects this service. But I simplified this way a couple of services as you suggested, for example UserTransactionRegistryService or UserTransactionService - see previous commits Brian.